web/src/BeaconPanel.jsx

```
@@ -0,0 +1,361 @@
+              {body.length}
+            </div>
+          )}
+        </div>
+        <button
+          onClick={handleSendMessage}
+          disabled={isSending}
+          className="primary-btn"
+          title="Send (Ctrl+Enter)"
+          aria-busy={isSending}
+        >
+          {isSending ? (
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)